pages/web/domains/domain_errors/guide.pl-pl.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Usunięcie błędu dla domeny'
+slug: domena-errors
+section: Zadania bieżące
+order: 01
+---
+
+> [!primary]
+> Tłumaczenie zostało wygenerowane automatycznie przez system naszego partnera SYSTRAN. W niektórych przypadkach mogą wystąpić nieprecyzyjne sformułowania, na przykład w tłumaczeniu nazw przycisków lub szczegółów technicznych. W przypadku jakichkolwiek wątpliwości zalecamy zapoznanie się z angielską/francuską wersją przewodnika. Jeśli chcesz przyczynić się do ulepszenia tłumaczenia, kliknij przycisk „Zaproponuj zmianę” na tej stronie.
+>
+
+**Ostatnia aktualizacja z dnia 01-09-2022**
+
+## Wprowadzenie
+
+Rejestracja domeny, jej transfer, zmiana właściciela to operacje, w których może wystąpić błąd. Może zaistnieć konieczność interwencji z Twojej strony.
+
+**Dowiedz się, jak postępować w przypadku wystąpienia błędu dla domeny.**
+
+## Wymagania początkowe
+
+- Posiadanie jednej lub kilku domen
+- Zalogowanie do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl)
+- Aktualizacja w [płatności](https://docs.ovh.com/pl/billing/zarzadzanie-fakturami-ovhcloud/#pay-bills) i [odnowienie](https://docs.ovh.com/pl/billing/przewodnik_dotyczacy_opcji_automatycznego_odnawiania_uslug_w_ovh/#renewal-management) powiązanych usług (domena i hosting)
+## W praktyce
+
+W [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) przejdź do sekcji `Web Cloud`{.action}, a następnie `nazwy domen`{.action}. Kliknij `Operacje w toku`{.action} nad listą domen.
+
+W tabeli możesz sprawdzić operacje związane z domenami w Twoim Panelu klienta.
+
+![domena](images/domain-error-table01.png){.thumbnail}
+
+- `Domena`: Domena, której dotyczy operacja.
+- `Operacja`:  Trwa wykonywanie operacji dla domeny.
+- `Komentarz`: Szczegóły operacji w trakcie. Instrukcje.
+- `Data przetwarzania`: Data utworzenia operacji.
+- `Data aktualizacji`:  Znacznik czasu aktualizacji operacji w trakcie.
+- `Data zakończenia`: Data zakończenia operacji.
+- `Status`: Obecny stan operacji.
+
+Wszystkie operacje wymienione w tej tabeli nie wymagają interwencji, aby były przeprowadzane w normalnych warunkach.<br>
+W niniejszym przewodniku przyjrzymy się operacjom **z błędem**, posługując się przykładami, które się powtarzają.
```

Comment:
Please check if bold phrase should contain also "operacjom"

Reply:
z błędem = error > this is good